src/mission_start.cpp

```diff
@@ -1748,7 +1704,7 @@ void reveal_route( mission *miss, const tripoint destination )
 {
     const npc *p = g->find_npc( miss->get_npc_id() );
     if( p == nullptr ) {
-        debugmsg( "mission_start::infect_npc() couldn't find an NPC!" );
+        debugmsg( "mission_start::reveal_route() couldn't find an NPC!" );
```

Comment:
Btw. putting the function name into the message string is redundant. The `debugmsg` macro will automatically include it anyway.

Reply:
Not really my code, but cleaned it up anyway.